app/ts/store/transaction.ts

```diff
@@ -0,0 +1,109 @@
+import { signal } from '@preact/signals'
+import { ethers } from 'ethers'
+import { assertsExternalProvider } from '../library/utilities.js'
+import { TransactionReceipt, TransactionResponse } from '../types.js'
+
+export type SendFnInput = {
```

Comment:
```suggestion
export type SendFunctionInput = {
```

avoid abbreviations

Reply:
addressed in 05ebe3fdcba955e415de5cdc13c7a8c5336c7094